mods/voidmulti.sc2mod/base.sc2data/gamedata/actordata.xml

```diff
@@ -13,6 +13,23 @@
     <CActorAction id="AdeptAttack">
         <ImpactModel value="AdeptAttackImpactModel"/>
     </CActorAction>
+    <CActorUnit id="Hydralisk">
+        <On index="65" Send="AnimBracketStart AttackSuperior Attack,Superior {} {} 0 1.245509 AsTimeScale"/>
+        <On index="66" Send="AnimBracketStart AttackInferior Attack,Inferior {} {} 0 1.245509 AsTimeScale"/>
+        <On index="67" Send="AnimPlay AttackCover Attack,Cover 0 -1.000000 -1.000000 1.500000 AsTimeScale"/>
+        <On index="68" Send="AnimBracketStart Attack Attack {} {} 0 1.245509 AsTimeScale"/>
+        <On index="73" Send="TimerSet 0.267000 MeleeAttackLaunch"/>
+        <On index="74" Send="TimerSet 0.066700 MeleeAttackLaunch"/>
+    </CActorUnit>
+    <CActorAction id="MutaliskAttackSegment2">
+        <LaunchAssets Sound=""/>
+    </CActorAction>
+    <CActorAction id="MutaliskAttackSegment3">
+        <LaunchAssets Sound=""/>
+    </CActorAction>
+    <CActorRange id="SensorTowerRadar">
+        <Range value="27.000000"/>
```

Comment:
Sensor Tower's Radar range reduced from 30 to 27

This changes the range indicator's size